Documentation/installation/cni-chaining-azure-cni.rst

```diff
@@ -21,6 +21,12 @@ Azure CNI (Legacy)
 
 .. include:: cni-chaining-limitations.rst
 
+.. admonition:: Video
+ :class: attention
+
+  If you'd like a video explanation of the Azure CNI Powered by Cilium, check out `eCHO episode 70: Azure CNI Powered by Cilium <https://www.youtube.com/live/8it8Hm2F_GM?si=E_z57MLGJZeZJSyN>`__.
```

Comment:
I'm also noting that this link doesn't have the same format as the others (`.../live/...` instead of `.../watch?v=...`), is this on purpose?

Reply:
Not really, its a youtube thing for live videos, it depends on how you access the video. If it's from the playlist then it'll use the latter format but if its from the live streams tab, it'll use the former. However, I've updated the link to use the latter format